pkg/build

```diff
@@ -5,59 +5,32 @@ srcdir ?= .
 AM_DEFAULT_VERBOSITY ?= 0
 NODE_ENV ?= production
 
-WEBPACK_PACKAGES = \
-	base1 \
-	apps \
-	kdump \
-	metrics \
-	networkmanager \
-	packagekit \
-	playground \
-	selinux \
-	shell \
-	sosreport \
-	static \
-	storaged \
-	systemd \
-	users \
-	$(NULL)
+# one built file in dist/ which we use as dependency
+DIST_STAMP = $(srcdir)/dist/static/manifest.json
 
-MANIFESTS = $(WEBPACK_PACKAGES:%=$(srcdir)/dist/%/manifest.json)
+# dynamic pkg → dist dependency, to rebuild the webpack if any web related file changes
+# exclude automake unit test log files
+PKG_INPUTS = $(shell find $(srcdir)/pkg -type f ! -name 'test*.trs' ! -name 'test*.log')
```

Comment:
More objections about this.  Do you want to land this, or should we wait for the changes where `webpack-make` will scan the timestamps itself?

Reply:
I have a version where all of this goes away. But given how hard it is to get this PR to green (affected retries, and other details), I'd prefer to land this as a follow-up.